utils/.gitignore

```diff
@@ -0,0 +1,37 @@
+HELP.md
```

Comment:
You don't need to repeat `.gitignore` in each module. Just place it once in the root.